src/shared/ui/survey/StackedCheckboxItem/StackedCheckboxItem.tsx

```diff
@@ -0,0 +1,54 @@
+import { FC } from 'react';
+import { StyleSheet } from 'react-native';
+
+import StackedCheckboxConfig from './types';
+import { CheckBox } from '../..';
+import { StackedItemsGrid } from '../StackedItemsGrid';
+import { StackedRowItemValue } from '../StackedItemsGrid/types';
+
+type Props = {
+  value: Array<string[]>;
+  onChange: (value: Array<string[]>) => unknown;
+  config: StackedCheckboxConfig;
+};
+
+const StackedCheckboxItem: FC<Props> = ({ value = [], onChange, config }) => {
```

Comment:
value = [],
this has to be required prop, I think there is no need for default value.

Reply:
But what if user takes survey for a first time? There would be no value, right?

Reply:
@vmkhitaryanscn  `value` is already required according to `Props`.